src/services/doppler-api-client.ts

```diff
@@ -121,13 +128,22 @@ export class HttpDopplerApiClient implements DopplerApiClient {
     }));
   }
 
+  private mapLinks(data: any): Link[] {
+    return data.map((x: any) => ({
+      href: x.href,
+      description: x.description,
+      rel: x.rel,
+    }));
+  }
+
   private mapCampaignsDelivery(data: any): CampaignDelivery[] {
     return data.map((x: any) => ({
       campaignId: x.campaignId,
       campaignName: x.campaignName,
       campaignSubject: x.campaignSubject,
       clicksCount: x.clicksCount,
       deliveryStatus: x.deliveryStatus,
+      links: this.mapLinks(x._links),
```

Comment:
Have taken in account that maybe there could be no links?

Reply:
yes!

Reply:
In no links case it would be empty array. So it's ok.